src/SFA.DAS.AssessorService.Application.Api.External/Controllers/CertificateController.cs

```diff
@@ -0,0 +1,286 @@
+
+namespace SFA.DAS.AssessorService.Application.Api.External.Controllers
+{
+    [Route("api/v1/certificates")]
+    [ApiController]
+    public class CertificateController : ControllerBase
+    {
+        private readonly ILogger<CertificateController> _logger;
+        private readonly IHeaderInfo _headerInfo;
+        private readonly ApiClient _apiClient;
+
+        public CertificateController(ILogger<CertificateController> logger, IHeaderInfo headerInfo, ApiClient apiClient)
+        {
+            _logger = logger;
+            _headerInfo = headerInfo;
+            _apiClient = apiClient;
+        }
+
+        [HttpPut(Name = "Put")]
+        [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(List<CertificateResponse>))]
+        [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
+        public async Task<IActionResult> CreateDraft([FromBody] List<CertificateRequest> request)
+        {
+            List<CertificateResponse> response = new List<CertificateResponse>();
+
+            foreach (var validatedCertificate in await ValidateCertificateRequest(request, _headerInfo.Ukprn, _headerInfo.Username))
+            {
+                CertificateResponse certificateResponse = new CertificateResponse
+                {
+                    Uln = validatedCertificate.Uln,
+                    LastName = validatedCertificate.LastName,
+                    StdCode = validatedCertificate.StdCode,
+                    CertificateData = validatedCertificate.CertificateData,
+                    Certificate = null,
+                    Status = CertificateStatus.Error,
+                    ValidationErrors = new List<string>()
+                };
+
+                if (!validatedCertificate.IsVaild)
+                {
+                    certificateResponse.ValidationErrors.AddRange(validatedCertificate.ValidationErrors);
+                }
+                else
+                {
+                    var scr = new AssessorService.Api.Types.Models.Certificates.StartCertificateRequest { Uln = validatedCertificate.Uln, StandardCode = validatedCertificate.StdCode, UkPrn = _headerInfo.Ukprn, Username = _headerInfo.Username };
+
+                    Certificate startCertificate = await _apiClient.StartCertificate(scr);
+
+                    CertificateResponse updateCertificateResponse = await UpdateDraftCertificate(startCertificate, validatedCertificate.CertificateData, _headerInfo.Username, CertificateStatus.Draft, CertificateActions.Start);
+
+                    certificateResponse.Certificate = updateCertificateResponse.Certificate;
+                    certificateResponse.Status = updateCertificateResponse.Status;
+                    certificateResponse.ValidationErrors.AddRange(updateCertificateResponse.ValidationErrors);
+                }
+
+                response.Add(certificateResponse);
+            }
+
+            return Ok(response);
+        }
```

Comment:
These controllers are doing too much really.  Push as much as you can back into the handler in the internal API.  Both the external and internal API layers should be as thin and dumb as possible. 😁